### PR TITLE
use secret team as code owners to avoid review requests spam.

### PR DESCRIPTION
**Pull request by Feiyang1**
The PR addresses 2 issues:
1. the current CODEOWNERS setup is too rigid and it doesn't work in case like cross package changes and doesn't provide an escape hatch when owners are not available.
2. Automatic review requests are spammy for large teams.

### CODE REVIEW

File: packages/testing/src/api/index.ts [hunk 1]
```diff
@@ -109,10 +109,10 @@ function initializeApp(
 ): firebase.app.App {
   let appOptions = {};
   if (databaseName) {
-    appOptions["databaseURL"] = `http://${DATABASE_ADDRESS}?ns=${databaseName}`;
+    appOptions['databaseURL'] = `http://${DATABASE_ADDRESS}?ns=${databaseName}`;
   }
   if (projectId) {
-    appOptions["projectId"] = projectId;
+    appOptions['projectId'] = projectId;
```
Comment (mikelehen) [inline packages/testing/src/api/index.ts]: These changes seem unrelated...?  (but if that's what prettier wants, 🤷‍♂️ )

**Comment by Feiyang1** [comment]
@bojeil-google, The global approver team has 2 purposes:
1. act as a fallback group when the owner(s) of a component is not available, e.g. on vacation. It's especially useful for small teams like rxFire, storage.
2. make cross package changes, large scale refactoring easier to get approved. E.g. library version upgrades, packaging changes, etc. 

The situation you mentioned could happen, but as a principle, we trust people do the right thing.

**Comment by Feiyang1** [comment]
It won't be added automatically. Only visible owners will be added.
This is actually the reason why secret team is used - Firestore team doesn't want to receive any review request automatically.
